site/content/resources/upgrading.md

```diff
@@ -9,9 +9,68 @@ This document describes the changes needed to upgrade your Contour installation.
 
 <div id="toc" class="navigation"></div>
 
-## Upgrading Contour 1.17.0 to 1.17.1
+## Upgrading Contour 1.17.1 to 1.18.0
+
+Contour 1.18.0 is the current stable release.
+
+### Required Envoy version
+
+All users should ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.19.0`.
+
+Please see the [Envoy Release Notes][33] for information about issues fixed in Envoy 1.19.0.
+
+### The easy way to upgrade
+
+If the following are true for you:
+
+* Your installation is in the `projectcontour` namespace.
+* You are using our [quickstart example][18] deployments.
+* Your cluster can take a few minutes of downtime.
+
+Then the simplest way to upgrade to 1.18.0 is to delete the `projectcontour` namespace and reapply one of the example configurations:
+
+```bash
+$ kubectl delete namespace projectcontour
+$ kubectl apply -f {{< param base_url >}}/quickstart/v1.18.0/contour.yaml
+```
+
+This will remove the Envoy and Contour pods from your cluster and recreate them with the updated configuration.
+If you're using a `LoadBalancer` Service, (which most of the examples do) deleting and recreating may change the public IP assigned by your cloud provider.
+You'll need to re-check where your DNS names are pointing as well, using [Get your hostname or IP address][12].
+
+### The less easy way
```

Comment:
there was at least one CRD change as well to add `HTTPProxy.Spec.IngressClassName` so should reapply those too.

Reply:
good catch!